File: 2016/fall/src8/week1/cough4.py
def main():
    cough(3)
    sneeze(3)

def cough(n):
    say("cough", n)

def say(word, n):
    for i in range(n):
        print(word)

def sneeze(n):
    say("achoo", n)

if __name__ == "__main__":
    main()

File: 1/cough4.c
#include <cs50.h>
#include <stdio.h>

void cough(int n);
void say(string word, int n);
void sneeze(int n);

int main(void)
{
    cough(3);
    sneeze(3);
}

void cough(int n)
{
    say("cough", n);
}

void say(string word, int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("%s\n", word);
    }
}

void sneeze(int n)
{
    say("achoo", n);
}
